### ISSUE TITLE
Repetitive Code for printing data to Terminal

### ISSUE THREAD

**Issue by Sameer-Mann** [t0xic0der/wisdom-cli#26]
Currently everywhere where something needs to be printed "click.echo(click.style("message1")" is being repeated and could be replaced by a simple function increasing readability of code.

**Comment by gridhead** [comment]
Certainly we would need that. Would you want to pick this up? @Sameer-Mann

**Comment by Sameer-Mann** [comment]
yes assign it to me

**Comment by Sameer-Mann** [comment]
@t0xic0der  currently the function "savehtml" does nothing 
![Screenshot 2020-10-30 at 10 36 27 PM](https://user-images.githubusercontent.com/42488833/97735480-64cc6980-1b00-11eb-9438-9605aa84779b.png)
Is this intentional or does it need to be completed?

**Comment by Sameer-Mann** [comment]
Is this way "https://github.com/Sameer-Mann/wisdom-cli/blob/main/main.py" okay?

**Comment by gridhead** [comment]
> @t0xic0der currently the function "savehtml" does nothing
> ![Screenshot 2020-10-30 at 10 36 27 PM](https://user-images.githubusercontent.com/42488833/97735480-64cc6980-1b00-11eb-9438-9605aa84779b.png)
> Is this intentional or does it need to be completed?

Yes, that is intentional. There's still some work that needs to be done before the section is called complete.

**Comment by gridhead** [comment]
> Is this way "https://github.com/Sameer-Mann/wisdom-cli/blob/main/main.py" okay?

Looks okay to me. Gotta see if it works just right when you make a PR.
